include/z64actor.h

```diff
@@ -146,8 +146,9 @@ typedef struct ActorShape {
 //
 #define ACTOR_FLAG_10 (1 << 10)
 
-//
-#define ACTOR_FLAG_ENKUSA_CUT (1 << 11)
+// A grass bushel (EN_KUSA) has been destroyed.
```

Comment:
"bushel"?

Reply:
Yes. You cant just say "a grass", it means like a bunch of grass clumped together, which is kinda like what the en_kusa actor is